deployability/modules/testing/testing.py

```diff
@@ -24,14 +29,17 @@ def run(cls, payload: InputPayload) -> None:
         cls._setup(ansible, extra_vars['working_dir'])
         cls._run_tests(payload.tests, ansible, extra_vars)
         if payload.cleanup:
+            logger.info("Cleaning up")
             cls._cleanup(ansible, extra_vars['working_dir'])
 
     @classmethod
     def _get_extra_vars(cls, payload: InputPayload) -> ExtraVars:
         if not payload.dependencies:
+            logger.debug("No dependencies found")
             return ExtraVars(**payload.model_dump())
         
         dependencies_ip = []
+        logger.debug("Dependencies found. Parsing...")
```

Comment:
```suggestion
        logger.debug("Dependencies found. Starting the analysis..")
```

Reply:
In this case, we don't analyze dependencies, we parse the incoming file and get the IP of the dependencies, so we get the list of dependencies with only the values we want